index.html

```diff
@@ -958,226 +961,112 @@ <h6 class="title">Quicklinks</h6>
                       lng: 0
                     },
                     styles: [{
-                      "elementType": "geometry",
```

Comment:
Why did you remove this?

Reply:
It is because they are the color theme for the map.

The map previously there had a blue color scheme which was not matching the website. So I changed it into a white and grey color scheme that now matches the website.

You can see it here : https://i.imgur.com/ZdVwqzE.png

Reply:
Okay cool, nice work! :smile: